src/src/com/microsoft/aad/adal/AuthenticationContext.java

```diff
@@ -1590,11 +1634,10 @@ private String getTokenHash(String token) {
         final String mRefreshToken;
         final String mKey;
         final boolean mMultiResource;
+        final boolean mFamilyToken;
```

Comment:
This should be String, currently you receive 1 and for another family you might return 2